.github/scripts/mr_publish_results.py

```diff
@@ -22,6 +26,66 @@
 }
 
 
+def transform_to_seconds(duration: str) -> float:
+    h_split = duration.split("h")
+    if len(h_split) == 1:
+        rest = h_split[0]
+        hours = 0
+    else:
+        hours = int(h_split[0])
+        rest = h_split[1]
+    m_split = rest.split("m")
+    if len(m_split) == 2:
+        minutes = int(m_split[0])
+        seconds = float(m_split[1].rstrip("s"))
+    elif len(m_split) == 1:
+        minutes = 0
+        seconds = float(m_split[0].rstrip("s"))
+    else:
+        raise Exception(f"Unsupported duration: {duration}")
+    overall_seconds = hours * 60 * 60 + minutes * 60 + seconds
+    return overall_seconds
+
+
+def send_to_datadog(context):
+    # Initialize
+    tags = {
+        "dataset": os.environ["DATASET_NAME"],
+        "dataset_repository_branch": DATASET_REPOSITORY_BRANCH,
+        "external_dataset_repository": IS_EXTERNAL,
+        "config_repository": CONFIG_REPOSITORY,
+        "config_repository_branch": CONFIG_REPOSITORY_BRANCH,
+        "dataset_repository_branch": os.environ["DATASET_REPOSITORY_BRANCH"],
+        "dataset_commit": os.environ["DATASET_COMMIT"],
+        "workflow": os.environ["GITHUB_WORKFLOW"],
+        "config": os.environ["CONFIG"],
+        "pr_url": os.environ["PR_URL"],
+        "accelerator_type": os.environ["ACCELERATOR_TYPE"],
+        "github_run_id": os.environ["GITHUB_RUN_ID"],
+        "github_sha": os.environ["GITHUB_SHA"],
+        "github_event": os.environ["GITHUB_EVENT_NAME"],
+        "type": os.environ["TYPE"],
+        "branch": os.environ["BRANCH"],
+    }
+    tags_list = [f"{k}:{v}" for k, v in tags.items()]
+    options = {
+        "statsd_host": "127.0.0.1",
```

Comment:
why do you need it for? I assume you're using a DD agent to send metrics, it'll be better if you send the metrics via API. https://docs.datadoghq.com/api/latest/metrics/#submit-metrics